sleap/gui/commands.py

```diff
@@ -3901,3 +3905,19 @@ def copy_to_clipboard(text: str):
     clipboard = QtWidgets.QApplication.clipboard()
     clipboard.clear(mode=clipboard.Clipboard)
     clipboard.setText(text, mode=clipboard.Clipboard)
+
+class UnlinkVideo(EditCommand):
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        video = params.get("video", None)
+        cam = params.get("Camcorder", None)
+        recording_session = params.get("RecordingSession", None)
+
+        if video is not None and recording_session is not None \
+            and cam is not None:
```

Comment:
I think we should just handle this by disabling the Unlink Button if the GUI is not in the correct state.